packages/smooth_app/lib/cache/cache_manager.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/foundation.dart';
+
+/// Abstract class defining a cache containing items of type [T] and accessible
+/// via a unique key of type [K]
+abstract class CacheMap<K, T> extends ChangeNotifier {
```

Comment:
I can't remember why we need a `ChangeNotifier` now.
Couldn't we just call an `async` get?

Reply:
ChangeNotifier was for Provider, so yes removing it